pkg/kube/controllers/extendedjob/trigger_reconciler.go

```diff
@@ -119,13 +119,14 @@ func (r *TriggerReconciler) Reconcile(request reconcile.Request) (result reconci
 	return
 }
 
-func (r *TriggerReconciler) createJob(ctx context.Context, eJob ejv1.ExtendedJob, podName string) error {
+func (r *TriggerReconciler) createJob(ctx context.Context, eJob ejv1.ExtendedJob, podName string, podUID string) error {
 	template := eJob.Spec.Template.DeepCopy()
 
 	if template.Labels == nil {
 		template.Labels = map[string]string{}
 	}
 	template.Labels["ejob-name"] = eJob.Name
+	template.Labels["triggering-pod"] = podUID
```

Comment:
could we constantize label keys, such as: https://github.com/cloudfoundry-incubator/cf-operator/blob/d2c424b3ecd3acce4efdfe2b252dc9cb61cb11cd/pkg/kube/apis/extendedjob/v1alpha1/types.go#L17-L20

Reply:
Maybe it's a little bit beyond the scope of this PR to replace all the keys, but since we have to test it all again anyhow....